static/bidder-info/beyondmedia.yaml

```diff
@@ -1,4 +1,3 @@
-endpoint: "http://backend.andbeyond.media/pserver"
```

Comment:
The endpoint is required, unless if you want to change the adapter to be disabled by default.

Reply:
@SyntaxNode Thanks, updated